content/applications/inventory_and_mrp/manufacturing/basic_setup/mo_costs.rst

```diff
@@ -137,7 +154,9 @@ same costs. This is the *estimated* cost of completing the |MO|.
 
 However, once work commences, the values in the :guilabel:`Real Cost` column may begin to diverge
 from the values in the :guilabel:`MO Cost` column. This happens if a different component quantity is
-used than was listed on the |MO|, or if the duration of a work order is different than expected.
+used than was listed on the |MO|, the duration of a work order is different than expected, or the
+hourly cost of the employee performing a work order differs from the employee cost set on the work
+center.
 
 Once the |MO| has been completed by clicking :guilabel:`Produce All`, the values in the
 :guilabel:`MO Cost` column update to match those displayed in the :guilabel:`Real Cost` column.
```

Comment:
It's not letting me comment on the individual lines below, but the example you walk through is awesome! Very clearly explained, and I am not left with any questions.

A couple minor suggestions/questions:
- I notice in the bullet points that single digit numbers (e.g. seven) are spelled out, while two+ digit numbers (e.g. 15) use numbers. I am familiar with this practice as a journalistic/academic writing convention, but I feel like for something like Odoo, it is kind of awkward. Personally I would prefer to see all numbers, like 1, 15, 300, etc., instead of written out. But is it an Odoo style convention we have to adhere to?
- Fairway Fields is italicized at the beginning of the example, but not when mentioned later. Is that intentional? Same with the putting green.

Reply:
I've asked about the numbers before and been told that this is the correct formatting to use. I've also been told to only italicize things the first time they're mentioned.